tests/functional/modules/test_zos_job_submit_func.py

```diff
@@ -1113,3 +1157,65 @@ def test_zoau_bugfix_invalid_utf8_chars(ansible_zos_module):
             assert result.get("changed") is True
     finally:
         hosts.all.file(path=temp_path, state="absent")
+
+
+def test_job_submit_async(get_config):
+    # Creating temp JCL file used by the playbook.
+    tmp_file = tempfile.NamedTemporaryFile(delete=True)
+    with open(tmp_file.name, "w",encoding="utf-8") as f:
+        f.write(JCL_FILE_CONTENTS)
+
+    # Getting all the info required to run the playbook.
+    path = get_config
+    with open(path, 'r') as file:
+        enviroment = yaml.safe_load(file)
+
+    ssh_key = enviroment["ssh_key"]
```

Comment:
Just an FYI, this will only work with the `zinventory` fixture because a user populates Jenkins with an SSH key, while fixture `zinventory-raw` has no support SSH keys currently so this test can not be part of the Concurrent executor (CE.py).

To support ssh keys, CE.py can be updated to take that as an argument and then update `zinventory-raw` to handle it, but honestly don't feel that is the right solution. Keys already exists on the host that can be looked up and passed along without user intervention or added to the hosts file which is what users.py does (actually generates a key and then adds it and removes it).